src/order/filter/ennreal.lean

```diff
@@ -40,9 +40,13 @@ begin
   rwa [div_def, one_mul],
 end
 
-lemma limsup_eq_zero_iff [ne_bot f] [countable_Inter_filter f] {u : α → ennreal} :
+lemma limsup_eq_zero_iff [countable_Inter_filter f] {u : α → ennreal} :
   f.limsup u = 0 ↔ u =ᶠ[f] 0 :=
 begin
+  by_cases hf_bot: f.ne_bot,
```

Comment:
I think you can remove all these `by_cases : f.ne_bot` if instead of using `limsup_const` you use a lemma `limsup_zero` that says that `limsup` of the constant 0 is 0, without the `ne_bot` assumption.